webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-26T18:05:21Z" title="Thursday, March 26th 2020, 7:05:21 pm +01:00">Mar 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
+    <ds-space v-if="this.connections && this.connections.length" margin="x-small">
```


```suggestion
    <ds-space v-if="connections && connections.length" margin="x-small">
```
don't know how this isn't throwing an error, but you don't have access to this in the `html`